src/realm/group.hpp

```diff
@@ -334,6 +334,22 @@ class Group : public ArrayParent {
     bool has_table(StringData name) const noexcept;
     TableKey find_table(StringData name) const noexcept;
     StringData get_table_name(TableKey key) const;
+    static StringData table_name_to_class_name(StringData table_name)
+    {
+        REALM_ASSERT(table_name.begins_with(g_class_name_prefix));
+        return table_name.substr(g_class_name_prefix_len);
+    }
+    using TableNameBuffer = std::array<char, max_table_name_length>;
+    static StringData class_name_to_table_name(StringData class_name, TableNameBuffer& buffer)
+    {
+        constexpr const char class_prefix[] = "class_";
+        constexpr size_t class_prefix_len = sizeof(class_prefix) - 1;
```

Comment:
we can use `g_class_name_prefix` and `g_class_name_prefix_len` here instead of redefining them

Reply:
Of course.